usr/sbin/serviceAccessConfig

```diff
@@ -40,13 +40,15 @@ Options:
 
 import configparser
 import logging
+import os
+import serviceAccessConfig.generatorfactory as factory
```

Comment:
please move the custom imports below the Python module imports, i.e.

```
import sys

import serviceAccessConfig.generatorfactory as factory
```

Reply:
Fixed in ac15be0.